src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -16,44 +17,73 @@ import { ACTIVITY_TARGETS } from '../../../common/interactionTargets';
 import { PLACEHOLDER_USER } from '../../../../constants';
 import type { Annotation, AnnotationPermission } from '../../../../common/types/feed';
 import type { GetAvatarUrlCallback, GetProfileUrlCallback } from '../../../common/flowTypes';
-import type { User } from '../../../../common/types/core';
+import type { SelectorItems, User } from '../../../../common/types/core';
 
 import './AnnotationActivity.scss';
 
 type Props = {
     currentUser?: User,
     getAvatarUrl: GetAvatarUrlCallback,
+    getMentionWithQuery?: Function,
     getUserProfileUrl?: GetProfileUrlCallback,
     isCurrentVersion: boolean,
+    isDisabled?: boolean,
     item: Annotation,
+    mentionSelectorContacts?: SelectorItems<>,
```

Comment:
Should this have a type argument for `SelectorItems<>`?

Reply:
Yes we can have it take in `User`.